cpu/samd21/periph/uart.c

```diff
@@ -26,6 +26,7 @@
 #include "periph/uart.h"
 #include "periph/gpio.h"
 
+#if UART_NUMOF
```

Comment:
This should be `#ifdef`, throwing a compilation error for other samd21 based boards else.